cvat-core/src/api-implementation.ts

```diff
@@ -345,28 +345,24 @@ export default function implementAPI(cvat) {
         };
     };
 
-    cvat.organizations.acceptInvitation.implementation = async (
-        username,
-        firstName,
-        lastName,
-        email,
-        password,
-        userConfirmations,
-        key,
-    ) => {
+    cvat.organizations.acceptInvitation.implementation = async (key) => {
```

Comment:
Now the implementation here looks useless.
Will it work? (The same for rejectInvitation)

``cvat.organizations.acceptInvitation.implementation = serverProxy.organizations.acceptInvitation; ``

Reply:
It doesnt really work with Jest tests. If we dont use such implementation(even dummy). The error is thrown:
```
TypeError: Cannot read properties of undefined (reading 'acceptInvitation')
cvat.organizations.acceptInvitation.implementation = serverProxy.organizations.acceptInvitation;
```